docs/guide/writing-tasks.md

```diff
@@ -171,7 +171,7 @@ CIRRUS_BASE_SHA | Base SHA if current build was triggered by a PR
 CIRRUS_BRANCH | Branch name. For example `my-feature`
 CIRRUS_BUILD_ID | Unique build ID
 CIRRUS_CHANGE_IN_REPO | Git SHA
-CIRRUS_CHANGE_MESSAGE | Commit message
+CIRRUS_CHANGE_MESSAGE | Concatenation of the pull-request summary and description at the time the task was originally triggered
```

Comment:
For non PR's it will be a commit message though. Maybe change it to something like?

```suggestion
CIRRUS_CHANGE_MESSAGE | Commit message or PR title and description depending on an event which triggered the build
```